helpers/proxy.ts

```diff
@@ -0,0 +1,10 @@
+/* Function to conditionally load the global-agent/bootstrap module */
+export async function initializeGlobalAgent() {
+  if (process.env.GLOBAL_AGENT_HTTP_PROXY) {
+    /* Dynamically import global-agent/bootstrap */
+    require("global-agent/bootstrap");
```

Comment:
@mohdamir here are some reasons why it's not a good idea to use require in an ESM project: https://www.perplexity.ai/search/using-require-in-Imt.F6GRSo.XELpXxlCyXQ

Reply:
@marcusschiesser  Used import instead of required. That required type declaration as well for global-agent which is committed inside types folder.